tests/models/compute/security_group_tests.rb

```diff
@@ -35,6 +35,32 @@
       @group.ip_permissions.empty?
     end
 
+    test("authorize access at a port range (egress rule)") do
+      @group.authorize_port_range(5000..6000, :direction => 'egress')
+      @group.reload
+      ip_permissions_egress = @group.ip_permissions_egress.dup
+      ip_permissions_egress.keep_if do |permission|
```

Comment:
it would be easier (and 1.8.7 compatible) to use `@group.ip_permissions_egress.find { }` and allow that to be the assertion.